docs/eog/case-studies-cw20-outputs-and-outcomes-plan.rst

```diff
@@ -0,0 +1,52 @@
+.. _CW20-Outputs-and-Outcomes-Plan: 
+
+CW20 Outputs and Outcomes Plan
+===============================
+
+Outputs
+--------------------
+
+- Presentations
+
+    - From keynotes (`slides <https://cw20.figshare.com/>`_, `videos <https://www.youtube.com/playlist?list=PLpX1jXuNTXGrAZr9I-jo0TNYNpfa5gpji>`_)
+    - From lightning talks  (`slides <https://cw20.figshare.com/>`_, `videos <https://www.youtube.com/playlist?list=PLpX1jXuNTXGqsMGvMAQQSpCCpbwu1Th7a>`_)
+    - From mini-workshops and demo sessions (`videos <https://www.youtube.com/playlist?list=PLpX1jXuNTXGqYOz7idfOVp6KfBTrjWlWS>`_)
+
+- `Speed blog posts <https://software.ac.uk/tags/cw20-speed-blog-posts>`_ from the discussion sessions
+- `Collaborative Ideas <https://software.ac.uk/cw20/collaborative-ideas-group>`_
+- Hack Day
+
+    - `Information about entries and winners <https://software.ac.uk/cw20/hackday-entries-and-winners>`_
+    - `Presentations <https://www.youtube.com/playlist?list=PLpX1jXuNTXGpBBfpZ93Uwas7FimQC2l1H>`_
+
+- `Infrastructure resources <https://doi.org/10.6084/m9.figshare.c.5026400>`_
+- `Blog posts <https://software.ac.uk/tags/collaborations-workshop-2020>`_
+
+
+Outcomes
+--------------------
+
+As part of our post-event feedback survey, we asked participants how confident their understanding was of each of the CW20 themes in the context of their research/work, both before and after the event. 
+In general, participants were more confident about their understanding of Open Research, Data Privacy and Software Sustainability after attending CW20.
+
```

Comment:
For all of these before and after pairs it's worth saying what the average score was before and after - and then a %-age improvement - it just makes it easier to understand the shift in understanding - e.g. on average CW20 improved attendees confidence from 3.2/5 to 4.1/5 a huge 30% increase <-- note numbers are made up and not checked just here for illustration : )